app/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -19,6 +20,8 @@ class HomeAssistantApplication : Application(), GraphComponentAccessor {
 
         AndroidThreeTen.init(this)
         graph = Graph(this, 0)
+        // Start the sensor worker if they start the app. The only other place we start this a Boot BroadcastReceiver
+        SensorWorker.start(this)
```

Comment:
We only report sensor values if location reporting is on. I think with this change it will always report ? Also it looks like it was called inside `sendLocationUpdate` because it would restart the 15 minute interval. The idea was that it would send at least once every 15 minutes, but more often if other things cause data to be sent (like connected status changed via this PR)

Reply:
I like the idea of sending sensors even with location disabled personally.  That start method will send all available sensors then restart the 15 minute timer, it should be safe to call here.